app/src/main/java/io/github/droidkaigi/confsched2017/repository/contributors/ContributorsRepository.java

```diff
@@ -49,7 +49,7 @@
     }
 
 
-    private Single<List<Contributor>> findAllFromRemote() {
+    public Single<List<Contributor>> findAllFromRemote() {
```

Comment:
I think it's better to create `setIsDirty(boolean isDirty)` and call it before `findAll()`.
What do you think?